src/groups.jl

```diff
@@ -143,6 +143,15 @@ function Base.isfinite(G::Group)
 You need to implement `Base.isfinite(::$(typeof(G))) yourself."""))
 end
 
+
+@doc Markdown.doc"""
+    istrivial(G::Group)
+Test whether group $G$ is trivial.
+
+The default implementation is based on `isfinite` and `order`.
+"""
+istrivial(G::Group) = isfinite(G) && isone(order(BigInt, G))
```

Comment:
well then, we can improve a bit:
```julia
function istrivial(G::Group)
    hasgens(G) && return all(isone, gens(G))
    isfinite(G) && return isone(order(G))
    return false
end
```
hmm... there used to be "suggest" button but I can't find it now? anyway, I'm happy with this!